iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -79,105 +77,214 @@ static llvm::cl::opt<int> defaultWorkgroupTileSize(
         "linalg.generic and linalg.indexed_generic workgroup tile size"),
     llvm::cl::init(64));
 
-static Optional<int64_t> getNativeVectorSize(FuncOp entryPointFn,
-                                             Type elementType) {
-  Optional<int64_t> nativeVectorSizeInBytes = llvm::None;
-  if (auto variantOp =
-          entryPointFn->getParentOfType<IREE::HAL::ExecutableVariantOp>()) {
-    if (IREE::HAL::ExecutableTargetAttr targetAttr = variantOp.target()) {
-      if (auto config = targetAttr.getConfiguration()) {
-        if (auto nativeVectorSizeAttr =
-                config.getAs<IntegerAttr>("native_vector_size")) {
-          if (int64_t nativeVectorSizeVal = nativeVectorSizeAttr.getInt()) {
-            nativeVectorSizeInBytes = nativeVectorSizeVal;
-          }
-        }
-      }
+/// Looks for the `native_vector_size` attribute in the hal.executable.variant
+/// op.
+static Optional<int64_t> getNativeVectorSizeInBytes(FuncOp entryPointFn) {
+  auto variantOp =
+      entryPointFn->getParentOfType<IREE::HAL::ExecutableVariantOp>();
+  if (!variantOp) return llvm::None;
+  IREE::HAL::ExecutableTargetAttr targetAttr = variantOp.target();
+  if (!targetAttr) return llvm::None;
+  auto config = targetAttr.getConfiguration();
+  if (!config) return llvm::None;
+  auto nativeVectorSizeAttr = config.getAs<IntegerAttr>("native_vector_size");
+  if (!nativeVectorSizeAttr) return llvm::None;
+  int64_t nativeVectorSizeVal = nativeVectorSizeAttr.getInt();
+  if (!nativeVectorSizeVal) return llvm::None;
+  return nativeVectorSizeVal;
+}
+
+/// Return the type length in bytes. Looks throuigh all the interface binding
```

Comment:
nit & typo: `Returns`, `through`

```suggestion
/// Returns the type length in bytes. Looks through all the interface binding
```